app/src/main/java/de/tum/in/tca/ticketcheck/component/ticket/TicketsController.java

```diff
@@ -67,6 +66,19 @@ public void onFailure(@NonNull Call<List<AdminTicket>> call, @NonNull Throwable
         };
         TUMCabeClient.getInstance(context).getAdminTicketData(eventID, callback);
     }
+
+    public AdminTicket getTicketById(int ticketId) {
+        return adminTicketDao.getByTicketId(ticketId);
+    }
+
+    public void redeemTicket(int ticketHistory, Callback<TicketSuccessResponse> cb){
```

Comment:
Maybe rename ticketHistory to ticketId as this is the common name which is used on client side

Reply:
it's only a local parameter, but yeah I'll change it